Modix.Services/Moderation/ModerationService.cs

```diff
@@ -533,7 +546,18 @@ public async Task UnConfigureGuildAsync(IGuild guild)
         }
 
         /// <inheritdoc />
-        public async Task RescindInfractionAsync(InfractionType type, ulong subjectId, string reason = null)
+        public async Task RescindInfractionAsync(long infractionId,
+            string reason = null)
+        {
```

Comment:
Need to make sure we don't lose the user/permission checks

```suggestion
        {
            AuthorizationService.RequireAuthenticatedUser();
            AuthorizationService.RequireClaims(AuthorizationClaim.ModerationRescind);
```